### PR TITLE
Add manual region tags

### PR DESCRIPTION
**Pull request by PierrickVoulet**

### CODE REVIEW

File: examples/remarketing/add_customer_match_user_list.py [hunk 1]
```diff
@@ -191,6 +192,7 @@ def _add_users_to_customer_match_user_list(
         "Offline user data job with resource name "
         f"'{offline_user_data_job_resource_name}' has finished."
     )
+    # [END add_customer_match_user_list]
```
Comment (wihl) [inline examples/remarketing/add_customer_match_user_list.py:195]: move after line 300

File: examples/remarketing/add_merchant_center_dynamic_remarketing_campaign.py [hunk 2]
```diff
@@ -75,6 +76,7 @@ def main(
                 for field_path_element in error.location.field_path_elements:
                     print(f"\t\tOn field: {field_path_element.field_name}")
         sys.exit(1)
+        # [END add_merchant_center_dynamic_remarketing_campaign]
```
Comment (wihl) [inline examples/remarketing/add_merchant_center_dynamic_remarketing_campaign.py:79]: move after line 308

**Review by wihl — COMMENTED** [review]
in `advanced_operations/add_dynamic_search_ads.py`, if nesting is allowed, you can add `add_dynamic_search_ads_3` around lines 173-175 (so [Java isn't the only language](https://developers.google.com/google-ads/api/docs/dynamic-search-ads/create-dynamic-search-ads?hl=en#create_the_dsa))

File: examples/remarketing/add_customer_match_user_list.py (hunk 1 above)
Comment (PierrickVoulet) [reply examples/remarketing/add_customer_match_user_list.py:195]: This region will replace what is currently displayed for Python (last code snippet at the bottom of this [page](https://developers.google.com/google-ads/api/docs/migration/user-data-service.md#python).

I am happy to change this to what you suggest but prefer to double check that we are aligned.

**Comment by PierrickVoulet** [comment]
> in `advanced_operations/add_dynamic_search_ads.py`, if nesting is allowed, you can add `add_dynamic_search_ads_3` around lines 173-175 (so [Java isn't the only language](https://developers.google.com/google-ads/api/docs/dynamic-search-ads/create-dynamic-search-ads?hl=en#create_the_dsa))

Nesting is allowed. This PR is only about migrating existing includecodes so I prefer to keep it as-is. I hope to work on filling those gaps soon.

**Review by wihl — APPROVED** [review]

**Pull request merged**